test/test-dns-round-robin.coffee

```diff
@@ -0,0 +1,24 @@
+should   = require 'should'
```

Comment:
need to remove the unused imports

Reply:
Agreed. Remove should whenever you see it